.github/workflows/js-ci.yml

```diff
@@ -132,9 +132,6 @@ jobs:
       - run: pnpm --fail-if-no-match --filter ${{ env.WORKSPACE }} build
         working-directory: js
 
-      - run: pnpm --fail-if-no-match --filter ${{ env.WORKSPACE }} cy:check-types
```

Comment:
How are the types for the integration tests checked now? I don't believe that Playwright does this for us, see [their documentation](https://playwright.dev/docs/test-typescript).

Reply:
because the playwright api is much more simple I don't think we need extra checks

Reply:
It still does not guarantee that the code that is written is correct, using the TypeScript compiler to verify that will prevent regressions.

Reply:
Let's handle this as a follow up, if you can log an issue for it we can resolve this thread.